conda_build/metadata.py

```diff
@@ -350,16 +353,9 @@ def yamlize(data):
     try:
         return yaml.load(data, Loader=StringifyNumbersLoader)
     except yaml.error.YAMLError as e:
-        if "{{" in data:
-            try:
-                import jinja2
-
-                jinja2  # Avoid pyflakes failure: 'jinja2' imported but unused
-            except ImportError:
-                raise exceptions.UnableToParseMissingJinja2(original=e)
```

Comment:
Just to understand, why are we removing this here? This looks like something 3rd party users might be relying on to identify why this is failing to parse

Reply:
Similar to `yaml`/`pyyaml`, `jinja2` used to be an optional dependency but is now [listed as a requirement in `meta.yml`](https://github.com/conda/conda-build/commit/c538eeaf4c23d0c3959a44f881997865f32d7707). However, since this is more "cleanup" vs "remove `sys.exit`", I will revert this change and leave it for the [original PR](https://github.com/conda/conda-build/pull/5255).